2020/hamming-codes/hungarian/sentence_translations.json

```diff
@@ -1027,7 +1027,7 @@
  },
  {
   "input": "Just forget about that zeroth bit entirely.",
-  "translatedText": "Felejtsd el teljesen a nulladik részt.",
+  "translatedText": "Felejtsd el teljesen a nulladik bit részt.",
```

Comment:
```suggestion
  "translatedText": "Szüntessük meg teljesen a nulladik bitet.",
```

Reply:
I admit this could be phrased better, but the correction suggests to "eliminate the 0th bit entirely", while the original phrasing refers to "let's forget it for now, will talk about it later". I'll come up with a better translation.